blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -174,8 +174,13 @@
     // 3 days - this matches the TTL for our metrics_full table, we don't accept anything older than the TTL.
     BEFORE_CURRENT_COLLECTIONTIME_MS("259200000"),
     // 10 minutes
-    AFTER_CURRENT_COLLECTIONTIME_MS("600000");
+    AFTER_CURRENT_COLLECTIONTIME_MS("600000"),
 
+    // Cross-Origin Resource Sharing
+    CORS_ALLOWED_ORIGINS(""),
```

Comment:
@shintasmith 
Good points... I'll modify the code to not create any CORS headers in the response if the origins are set to empty string.  It'll be the main switch for turning CORS support or and off.  

But since we're here, should we default this value to empty string in blueflood?  or should it be "*"?  With wiring the getEvents code to this configuration with this PR, it really should be "*" (which I was going to create a blueflood-chef PR to add to the blueflood.conf on the query nodes)... but I was afraid that "*" as the default might be a security risk for us and any installer of bluefloods if they're not aware of this setting.   What do you think the default should be?